src/ASTBuilder.ts

```diff
@@ -1386,7 +1388,7 @@ export class ASTBuilder
       const fragments = ctx
         .stringLiteral()!
         .StringLiteralFragment()
-        .map((stringLiteralFragmentCtx: any) => {
+        .map((stringLiteralFragmentCtx: ParseTree) => {
```

Comment:
`ctx.stringLiteral()!.StringLiteralFragment()` returns a `TerminalNode[]` so we can assume that the map will iterate over `ParseTree` since `TerminalNode` implements `ParseTree`

Reply:
Same comment, we can just use the inferred type here.